### PR TITLE
Implement numpy.arctanh function

### PR DESCRIPTION
**Pull request by Madhav2310**

### CODE REVIEW

**Comment by Madhav2310** [comment]
Please review @czgdp1807

File: integration_tests/elemental_11.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    observed1d_1 = reshape(identity1, newshape)
+    observed1d_2 = reshape(identity2, newshape)
+
+    for i in range(400):
+        assert abs(observed1d_1[i] - 0.0) <= eps
+        assert abs(observed1d_2[i] - 0.0) <= eps
+
+
+elemental_arctanh()
+elemental_trig_identity()
```
Comment (czgdp1807) [inline integration_tests/elemental_11.py:85]:
```suggestion


elemental_arctanh()
elemental_trig_identity()

```

File: src/runtime/lpython_intrinsic_numpy.py [hunk 2]
```diff
@@ -294,3 +294,23 @@ def radians(x: f64) -> f64:
...
+    return _lfortran_datanh(x)
+
+@ccall
+def _lfortran_satanh(x: f32) -> f32:
+    pass
+
+@overload
+@vectorize
+def arctanh(x: f32) -> f32:
+    return _lfortran_satanh(x)
```
Comment (czgdp1807) [inline src/runtime/lpython_intrinsic_numpy.py:316]:
```suggestion
@overload
@vectorize
def arctanh(x: f32) -> f32:
    return _lfortran_satanh(x)

```

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

**Review by czgdp1807 — APPROVED** [review]
@Smit-create Please check if this is good to go. Though wait for #1091 to be merged.

**Review by Smit-create — APPROVED** [review]

**Pull request merged**